modules/lti/src/utils.ts

```diff
@@ -1,5 +1,75 @@
 import { parse as parseQuery } from "query-string";
+import { Track } from "./OpencastRest";
 
 export function parsedQueryString() {
     return parseQuery(window.location.search);
 }
+
+/**
+ * Returns the given string, with the first letter capitalized
+ * @param s the string to capitalize
+ */
+export function capitalize(s: string) {
+  if (typeof s !== 'string') {
+    return '';
+  }
+  return s.charAt(0).toUpperCase() + s.slice(1);
+}
+
+/**
+ * Sorts an array of tracks alphabetically by their type
+ * If tracks have the same type, they are then sorted by resolution
+ * @param tracks the tracks to be sorted
+ */
+export const sortByType = (tracks: Track[]) => {
+  tracks.sort(function(a, b){
+    const nameA = a.type,
+        nameB = b.type;
+    const descA = a.resolution,
+        descB = b.resolution;
+
+    if(nameA > nameB) {
+      return 1;
+    }
+    if(nameB > nameA) {
+      // eslint-disable-next-line @typescript-eslint/no-magic-numbers
+      return -1;
+    }
+
+    if (descA === undefined || descB === undefined) {
+      return 0;
+    } // Else
+    return sortResolutions(descA, descB);
+
+  })
+  return tracks;
+}
+
+/**
+ * Helper function for sortByType. Sorts a resolution string by the number it contains
+ * @param descA resolution of the first track
+ * @param descB resolution of the second track
+ */
+const sortResolutions = (descA: string, descB: string) => {
```

Comment:
Either also move this function to trackUtils or make it a general function for comparing two resolutions. Also I would prefer to rename it compareResolutions, as it does not sort them, like the sortTracks above